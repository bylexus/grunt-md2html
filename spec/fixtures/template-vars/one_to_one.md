* Basepath: <%= basepath %>
* Dest: <%= destination %>
* Src: <%= src %>