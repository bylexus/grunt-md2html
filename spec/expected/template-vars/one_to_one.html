<ul>
<li>Basepath: my/base</li>
<li>Dest: tmp/template-vars/one_to_one.html</li>
<li>Src: spec/fixtures/template-vars/one_to_one.md</li>
</ul>
